### PR TITLE
[398] Fix Token Type Misspelling

### PR DESCRIPTION
**Pull request by luongthomas**
## What changes does this PR introduce?
This PR introduces a minor fix for token_types.

## Any background context you want to provide?
There was a missing `r` for the token_type `email_confirm`.  This prevented any additional email confirmation emails from being resent.  

## Where should the reviewer start?
`app/routes/account.py`

## Has this been manually tested? How?
Locally via Postman

## What value does this provide to our end users?
When they request a new email confirmation, then it should send one.

### CODE REVIEW

**Comment by luongthomas** [comment]
[Fix misspelling for the expected token_type when resending activation email](https://trello.com/c/iSOVCExL/398-fix-misspelling-for-the-expected-tokentype-when-resending-activation-email)

File: tests/functional/routes/test_account.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def test_obtain_email_confirm_token(
         )
 
         assert res.status_code == 200
-        send_confirm_email.assert_called_once()
+        assert send_confirm_email.call_count is 2
```
Comment (AndrewScibek) [inline tests/functional/routes/test_account.py]: Why is this called twice now?

Comment (luongthomas) [reply tests/functional/routes/test_account.py]: The first one is sent when we added an unconfirmed user in lines 18-20

Comment (luongthomas) [reply tests/functional/routes/test_account.py]: The second is from the post request

**Pull request merged**